File: app/src/main/java/com/jobtick/activities/ProfileActivity.java
package com.jobtick.activities;

import androidx.fragment.app.FragmentTransaction;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.MenuItem;

import com.google.android.material.appbar.MaterialToolbar;
import com.jobtick.R;

import com.jobtick.fragments.ProfileViewFragment;

import butterknife.BindView;
import butterknife.ButterKnife;

public class ProfileActivity extends ActivityBase {

    @SuppressLint("NonConstantResourceId")
    @BindView(R.id.toolbar)
    MaterialToolbar toolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_porfile);
        ButterKnife.bind(this);

        if(getIntent().getIntExtra("id",-1)!=-1){
            Bundle b = new Bundle();
            b.putInt("userId",getIntent().getIntExtra("id",-1));
            FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
            ProfileViewFragment profileFragment = new ProfileViewFragment();
            profileFragment.setArguments(b);
            ft.replace(R.id.profile, profileFragment);
            ft.commit();
        }

        initToolbar();
    }


    private void initToolbar() {
        toolbar.setNavigationIcon(R.drawable.ic_back);
        setSupportActionBar(toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setTitle("Profile");
    }


    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            onBackPressed();
        }

        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/com/jobtick/android/activities/ProfileActivity.kt
package com.jobtick.android.activities

import android.os.Bundle
import android.view.MenuItem
import com.google.android.material.appbar.MaterialToolbar
import com.jobtick.android.R
import com.jobtick.android.fragments.ProfileViewFragment

class ProfileActivity : ActivityBase() {
    var toolbar: MaterialToolbar? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_porfile)
        if (intent.getIntExtra("id", -1) != -1) {
            val b = Bundle()
            b.putInt("userId", intent.getIntExtra("id", -1))
            val ft = supportFragmentManager.beginTransaction()
            val profileFragment = ProfileViewFragment()
            profileFragment.arguments = b
            ft.replace(R.id.profile, profileFragment)
            ft.commit()
        }
        initToolbar()
    }

    private fun initToolbar() {
        toolbar = findViewById(R.id.toolbar)
        toolbar!!.setNavigationIcon(R.drawable.ic_back)
        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.title = "Profile"
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            onBackPressed()
        }
        return super.onOptionsItemSelected(item)
    }
}
